Makefile

```diff
@@ -123,7 +126,7 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: test
```

Comment:
I don't think we need to test the code every time we want to make a new local mage. Most of the times this command is being run because a local image is going to be tested manually.